satellite/marketing/marketingweb/server.go

```diff
@@ -42,7 +45,29 @@ type Server struct {
 	}
 }
 
-// commonPages returns templates that are required for everything.
+// Struct used to render each offer table
+type offerSet struct {
+	RefOffers,FreeCredits []marketing.Offer
+}
+
+func init(){
+	decoder.RegisterConverter(time.Time{}, timeConverter)
+}
+
+// Organizes offers from database by type.
+func organizeOffers(offers []marketing.Offer) offerSet{
+	var os offerSet
+	for _,offer := range offers {
+		if offer.Type == marketing.FreeCredit {
```

Comment:
Do you think that a `switch` statement here would have a better readability?

Reply:
Yes. Pushed.